core/domain/wipeout_service.py

```diff
@@ -240,19 +283,60 @@ def _generate_activity_to_pseudonymized_ids_mapping(activity_ids):
     }
 
 
-def _delete_models(user_id, module_name):
-    """Delete all the models from the given module, for a given user.
```

Comment:
Do not delete this method.

Reply:
Yeah, sorry, I have put it back. (Slightly misinterpreted its purpose why removed it earlier. Makes sense to me now!)